analyzers/src/SonarAnalyzer.CFG/ShimLayer/AnalysisContext/SymbolStartAnalysisContextWrapper.cs

```diff
@@ -76,6 +76,10 @@ static SymbolStartAnalysisContextWrapper()
         // receiverParameter => ((symbolStartAnalysisContextType)receiverParameter)."propertyName"
         Func<object, TProperty> CreatePropertyAccessor<TProperty>(string propertyName)
         {
+            if (symbolStartAnalysisContextType == null)
+            {
+                return static _ => default;
+            }
             var receiverParameter = Parameter(typeof(object));
             return Lambda<Func<object, TProperty>>(
```

Comment:
The `Lambda()` invocation throws an `ArgumentException` if `symbolStartAnalysisContextType` is null.